app/controllers/commontator/comments_controller.rb

```diff
@@ -216,6 +216,10 @@ def update_unread_status
           return
```

Comment:
I missed this in the last PR. Why do we need a global variable here (`@reader`)? Instead, pass the reader to `unseen_comments` as parameter. This way it's even easier to understand what is going on. A smiliar scenario can be found in the `update` method in `readers_controller`.